staging/src/k8s.io/component-base/config/v1alpha1/defaults.go

```diff
@@ -44,6 +44,7 @@ func RecommendedDefaultLeaderElectionConfiguration(obj *LeaderElectionConfigurat
 		obj.RetryPeriod = metav1.Duration{Duration: 2 * time.Second}
 	}
 	if obj.ResourceLock == "" {
+		// TODO: Migrate to LeaseLock.
```

Comment:
I think we would only migrate in the next version (v1beta1, etc), so this TODO here might be misleading to people looking for cleanup work to do

Reply:
Yeah - good point. Didn't think about it.
I hope noone will come quickly to fix it - if so, I will update the TODO.